src/EditorFeatures/CSharpTest/ConvertToInterpolatedString/ConvertPlaceholderToInterpolatedStringTests.cs

```diff
@@ -1035,6 +1033,22 @@ void M(ILogger logger)
     {
         logger.Log($""{5}"");
     }
+}");
+        }
+
+        [Fact, WorkItem(61346, "https://github.com/dotnet/roslyn/issues/61346")]
+        public async Task TestNotWithExplicitCultureInfo()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"using System;
+using System.Globalization;
+
+class T
+{
+    void M()
+    {
+        var a = string.Format(CultureInfo.InvariantCulture, [|""{0}"", 1|]);
```

Comment:
❔ Is this span expected? Or should it be on the containing `string.Format` call?

Reply:
Yup expected. Though the other span would work as well. The feature (predating my changes here) walks upwards, allowing selections on many levels.